### PR TITLE
Cosmetic Pr

### PR DESCRIPTION
**Pull request by Super3222**
#### Changelog

:cl:
rscadd: You can now knock over vending machines with your fists or with a weapon.
rscadd: You can pick vending machines back up by selecting them with help intent on.
rscadd: Soda cans require you to crack them open!
rscadd: Bagged snacks require you to pop them open!
rscadd: Computers make keyboard sounds!
fix: Increases the number of borer yummies in vendors to equal the regular number of snacks. (3 -> 6) 
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
unfinished op alert

File: code/modules/food_and_drinks/food/snacks_vend.dm [hunk 1]
```diff
@@ -29,6 +31,23 @@
...
+		if(!open)
+			user << "<span class='warning'>You haven't opened the bag yet!</span>"
+			return
+	return ..()
+
+/obj/item/weapon/reagent_containers/food/snacks/attack_self(mob/user)
+	if(requires_opening)
+		if(!open)
+			visible_message("<span class='notice'>[user] opens up a bag of chips.</span>",\
+							"<span class='notice'>[user] opens up a bag of chips.</span>")
```
Comment (ShadowDeath6) [inline code/modules/food_and_drinks/food/snacks_vend.dm]: it'd be better to refer to the actual object rather than "a bag of chips"

**Comment by Time-Green** [comment]
Nice, we should actually do more stuff like this

**Comment by drderp3635** [comment]
Cracking open a cold one with the boys

**Comment by Lightning38088** [comment]
What about flipping over tables

**Comment by Super3222** [comment]
> What about flipping over tables

All in good time

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -713,9 +786,9 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	product_slogans = "Try our new nougat bar!;Twice the calories for half the price!"
 	product_ads = "The healthiest!;Award-winning chocolate bars!;Mmm! So good!;Oh my god it's so juicy!;Have a snack.;Snacks are good for you!;Have some more Getmore!;Best quality snacks straight from mars.;We love chocolate!;Try our new jerky!"
 	icon_state = "snack"
-	products = list("Snacks" = list(/obj/item/weapon/reagent_containers/food/snacks/candy = 6,/obj/item/weapon/reagent_containers/food/drinks/dry_ramen = 6,/obj/item/weapon/reagent_containers/food/snacks/chips =6,
+	products = list("Snacks" = list(/obj/item/weapon/storage/byummie = 6, /obj/item/weapon/reagent_containers/food/snacks/candy = 6,/obj/item/weapon/reagent_containers/food/drinks/dry_ramen = 6,/obj/item/weapon/reagent_containers/food/snacks/chips =6,
```
Comment (Jc1469) [inline code/game/machinery/vending.dm:789]: addition of more products is a clear violation of balance, and is not documented either. i cannot allow this.
`/obj/item/weapon/storage/byummie = 6`
:balance_scale: :police_car: :balance_scale:

Comment (Super3222) [reply code/game/machinery/vending.dm:789]: updating OP

Comment (Jc1469) [reply code/game/machinery/vending.dm:789]: It was also 3 before, not 4.

**Comment by Mrcreepssss** [comment]
NO 6 BORER YUMMY'S? IN ONE  MACHINE? THAT IS WAY TOO POWERFULL YOU'LL DETROY THE WHOLE BALANCE OF THE GAME!!!

**Comment by matskuman5** [comment]
This is actually really cool, we need more immershun stuff like this

**Comment by Super3222** [comment]
:cry: Cruix is great

**Pull request merged**